resources/resinos.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation    This resource provides access to host OS.
+Library   Process
+Library   OperatingSystem
+Library   String
+Library   Collections
+
+*** Variables ***
+
+*** Keywords ***
+Set up loop device
+    ${result} =  Run Process    losetup -f -P /autohat/resin.img    shell=yes
```

Comment:
This keyword needs to take the resin.img path as an input - The aim is to have reusable resource files.